docs/scos/dev/front-end-development/202212.0/oryx/oryx-presets.md

```diff
@@ -37,11 +37,17 @@ export const b2cFeatures: AppFeature[] = [
   ...
 ```
 
+Feature sets also contain static experience data. Experience data includes the structure and layout of the components of the application, such as pages and sections. By utilizing the static experience data provided by the presets, you do no need to set up any boilerplate code. Moreover, we avoid hardcoded page structures that would not allow for personalized experiences going forward.
+
 For more information about feature sets, see [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/oryx-feature-sets.html)
 
 ## Themes
 
-The overarching UI of the application is driven by themes. To get started with a default theme, you can import it from the preset package and apply it to your Oryx application:
+Themes in Oryx play a key role in defining the visual appearance of your application. A theme represents the global typography, colors, and other specific design elements, such as form field placeholder color, that define the overall look and feel of your application.
+
+The theme consists of design tokens, which are configuration values that control the visual aspects of your application. Design tokens allow you to define global typography, colors, spacing, and other visual properties. By configuring these design tokens, you can customize the visual appearance of your application to align with your brand or specific design requirements.
```

Comment:
In those two paragraphs we are mentioning almost the same thing three times:
- "that define the overall look and feel of your application"
- "that control the visual aspects of your application"
- "customize the visual appearance of your application ..."

Not saying it's necesairly bad, but a bit noisy IMO.